docs/Intro/dive/onchain.md

```diff
@@ -6,12 +6,11 @@ keywords: [quorum, WatRegistry, ValidatorRegistry]
 ---
  # On-Chain Components
 
-    - **WatRegistry** is a Smart Contract designed to manage data models, particularly in the context of MultiSig (multi-signature) validation. Here's a breakdown of the information it stores for each data model:
 
     - **List of Validators**: This refers to the entities or individuals who are authorized to participate in the MultiSig validation process for a particular data model. MultiSig validation typically requires multiple parties to sign off on a transaction or data update before it is considered valid. The list of validators outlines who these authorized parties are.
     - **Quorum**: This represents the minimum number of validations or "feeds" required for a MultiSig validation to be considered valid for a specific data model. For example, if the quorum for a particular data model is set to 11, it means that exactly 11 out of the total number of validators listed must sign off on the transaction or data update for it to be accepted.
     
-     - By storing this information for each data model, **WatRegistry** ensures that the MultiSig validation process is conducted according to the specified requirements for each particular data model. Chronicle Protocol facilitates the dynamic updating of configurations for validators using the Chronicles Go-based Oracle client. **ConfigRegistry** assigns a unique URL to each individual validator. This unique URL mechanism allows for the management of distinct configurations for every validator. In essence, the **ConfigRegistry** serves as a central repository where configuration information for each validator is stored and can be updated as needed. By maintaining this registry, Chronicle Protocol ensures that each validator can have its own specific configuration settings tailored to its requirements, enhancing flexibility and customization within the protocol
+     - Chronicle Protocol facilitates the dynamic updating of configurations for validators using the Chronicles Go-based Oracle client. **ConfigRegistry** assigns a unique URL to each individual validator. This unique URL mechanism allows for the management of distinct configurations for every validator. In essence, the **ConfigRegistry** serves as a central repository where configuration information for each validator is stored and can be updated as needed. By maintaining this registry, Chronicle Protocol ensures that each validator can have its own specific configuration settings tailored to its requirements, enhancing flexibility and customization within the protocol.
```

Comment:
```suggestion
     - Chronicle Protocol facilitates the dynamic updating of configurations for validators using Chronicle's Go-based Oracle client. **ConfigRegistry** allows for the management of distinct configurations for every validator. In essence, the **ConfigRegistry** serves as a central repository where configuration information for each validator is stored and can be updated as needed. By maintaining this registry, Chronicle Protocol ensures that each validator can have its own specific configuration settings tailored to its requirements, enhancing flexibility and customization within the protocol.
```

Reply:
I am not sure that having different configs for different validators is a good thing, but it will be useful when we have a lot of validators and some will be dedicated to specific oracles.